### PR TITLE
Install rttest_plot and analyze.py as programs

### LINKED ISSUE #10: rttest_plot: installed without execute permissions
On Ubuntu Trusty installed using alpha1 binaries, the rtt_plot script has mode 0644.

### PR DESCRIPTION
**Pull request by esteve**
Use `install(PROGRAMS...)` to install the scripts with the right permissions.

Fixes #10

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -49,7 +49,7 @@ if(${ament_cmake_FOUND})
   )
 
   install(
-    DIRECTORY scripts/
+    PROGRAMS scripts/rttest_plot scripts/analyze.py
```
Comment (dirk-thomas) [inline CMakeLists.txt]: Alphabetical order.

**Comment by esteve** [comment]
@jacquelinekay does `analyze.py` have to be installed or it's just `rttest_plot` ?

**Comment by esteve** [comment]
@jacquelinekay I've removed the `analyze.py` script

File: CMakeLists.txt [hunk 2]
```diff
@@ -49,8 +49,8 @@ if(${ament_cmake_FOUND})
   )
 
   install(
-    DIRECTORY scripts/
-    DESTINATION bin
+    PROGRAMS scripts/rttest_plot
+    DESTINATION bin/rttest_plot
```
Comment (dirk-thomas) [inline CMakeLists.txt]: Have you tried this locally? I am pretty sure it will install the script into a subfolder under bin.

Comment (esteve) [inline CMakeLists.txt]: That's true, I was about to generate a tarball, thanks for pointing this out.

**Comment by esteve** [comment]
Packaging job:

http://ci.ros2.org/view/packaging/job/ros2_packaging_linux/33/

**Comment by esteve** [comment]
The packaging job generated a tarball with the correct permissions for `rttest_plot`

**Comment by dirk-thomas** [comment]
lgtm

**Pull request merged**